crates/pink/src/runtime.rs

```diff
@@ -114,6 +114,7 @@ parameter_types! {
     pub const DeletionWeightLimit: Weight = Weight::from_ref_time(500_000_000_000);
     pub const MaxCodeLen: u32 = 2 * 1024 * 1024;
     pub const MaxStorageKeyLen: u32 = 128;
+    pub const MaxDebugBufferLen: u32 = 2 * 1024 * 1024;
```

Comment:
Oh, good to see this config. With it, we can enable the `ink::debug_print!` and `panic!` now.
2MiB is kinda too big. Let's set it to 128 KiB.

Reply:
Changed to 128kiB